ports/servoshell/desktop/headed_window.rs

```diff
@@ -91,7 +91,9 @@ impl Window {
             .with_transparent(no_native_titlebar)
             .with_inner_size(LogicalSize::new(window_size.width, window_size.height))
             .with_min_inner_size(LogicalSize::new(1, 1))
-            .with_visible(true);
+            // Must be invisible at startup; accesskit_winit setup needs to
+            // happen before the window is shown for the first time.
+            .with_visible(false);
```

Comment:
Is this really true? I think I tested it without this change when I wrote https://github.com/servo/servo/issues/4344#issuecomment-2924804599, but it might be a platform specific thing again.

Reply:
It's documented in https://docs.rs/accesskit_winit/latest/accesskit_winit/struct.Adapter.html#method.with_event_loop_proxy and I believe it is platform-specific.